packages/ketcher-react/src/script/ui/dialog/template/template-attach.jsx

```diff
@@ -282,7 +302,7 @@ class Attach extends Component {
                 className={classes.button}
                 disabled={!this.checkIsValidName(name)}
               >
-                Apply
+                {this.mode === 'save' ? 'Save' : 'Edit'}
```

Comment:
You can make first letter uppercase with css rules:

```this.mode === this.MODES.SAVE  ? this.MODES.SAVE : this.MODES.EDIT```

Reply:
This part, `this.mode === this.MODES.SAVE`, is good. But for this part, `this.MODES.SAVE : this.MODES.EDIT`, it might be bad if we want to change the button text in the future